app/config/tstvhq/adventures.json

```diff
@@ -3,19 +3,19 @@
         {
             "config": {
                 "audio": {
-                    "path": "https://jitsi-party.s3.amazonaws.com/Satanic+Planet+999+(official+music+video).mp3",
+                    "path": "https://drive.google.com/file/d/142NtV56pgDTcQuBpHSUef6uJVDiqiStp/view?usp=sharing",
```

Comment:
Needs to be S3 URL. Send me a link to a file i can download myself and upload to S3. Can't figure out how to use that URL

Reply:
changed